--- build.nzbToMedia.git/nzbToHeadPhones.py	2017-02-06 11:43:40.041604001 -0600
+++ patches/nzbToHeadPhones.py	2017-02-06 11:53:51.444635785 -0600
@@ -46,7 +46,7 @@
 #hphost=localhost
 
 # HeadPhones port.
-#hpport=8181
+#hpport=8084
 
 # HeadPhones uses ssl (0, 1).
 #
@@ -61,7 +61,7 @@
 # HeadPhones watch directory.
 #
 # set this to where your HeadPhones completed downloads are.
-#hpwatch_dir=
+#hpwatch_dir=/storage/downloads/music
 
 # HeadPhones wait_for
 #
