--- build.nzbToMedia.git/nzbToCouchPotato.py	2017-02-06 11:43:40.041604001 -0600
+++ patches/nzbToCouchPotato.py	2017-02-06 11:52:54.408352957 -0600
@@ -51,7 +51,7 @@
 #cpshost=localhost
 
 # CouchPotato port.
-#cpsport=5050
+#cpsport=8083
 
 # CouchPotato uses ssl (0, 1).
 #
@@ -66,7 +66,7 @@
 # CouchPotato watch directory.
 #
 # set this to where your CouchPotato completed downloads are.
-#cpswatch_dir=
+#cpswatch_dir=/storage/downloads/movies
 
 # CouchPotato Postprocess Method (renamer, manage).
 #
