--- build.nzbToMedia.git/nzbToHeadPhones.py	2015-11-10 16:59:32.242193192 -0600
+++ patches/nzbToHeadPhonesnew.py	2015-11-10 17:00:58.262619743 -0600
@@ -40,7 +40,7 @@
 #hphost=localhost
 
 # HeadPhones port.
-#hpport=8181
+#hpport=8084
 
 # HeadPhones uses ssl (0, 1).
 #
@@ -55,7 +55,7 @@
 # HeadPhones watch directory.
 #
 # set this to where your HeadPhones completed downloads are.
-#hpwatch_dir=
+#hpwatch_dir=/storage/downloads/music
 
 # HeadPhones wait_for
 #
@@ -115,4 +115,4 @@
 
 section = "HeadPhones"
 result = nzbToMedia.main(sys.argv, section)
-sys.exit(result)
\ No newline at end of file
+sys.exit(result)
