--- build.nzbToMedia.git/core/__init__.py	2015-11-10 16:59:32.162192796 -0600
+++ patches/__init__.new.py	2015-12-09 21:41:44.105556779 -0600
@@ -7,17 +7,18 @@
 import time
 
 # init libs
-PROGRAM_DIR = os.path.dirname(os.path.normpath(os.path.abspath(os.path.join(__file__, os.pardir))))
+PROGRAM_DIR = '/storage/.kodi/userdata/addon_data/script.service.audo/scripts/'
 LIBS_DIR = os.path.join(PROGRAM_DIR, 'libs')
 sys.path.insert(0, LIBS_DIR)
+sys.path.insert(1, '/storage/.kodi/addons/script.module.audo-dependencies/lib')
 
 # init preliminaries
 SYS_ARGV = sys.argv[1:]
 APP_FILENAME = sys.argv[0]
 APP_NAME = os.path.basename(APP_FILENAME)
-LOG_DIR = os.path.join(PROGRAM_DIR, 'logs')
+LOG_DIR = '/storage/.kodi/userdata/addon_data/script.service.audo/logs/'
 LOG_FILE = os.path.join(LOG_DIR, 'nzbtomedia.log')
-PID_FILE = os.path.join(LOG_DIR, 'nzbtomedia.pid')
+PID_FILE = '/var/run/nzbtomedia.pid'
 CONFIG_FILE = os.path.join(PROGRAM_DIR, 'autoProcessMedia.cfg')
 CONFIG_SPEC_FILE = os.path.join(PROGRAM_DIR, 'autoProcessMedia.cfg.spec')
 CONFIG_MOVIE_FILE = os.path.join(PROGRAM_DIR, 'autoProcessMovie.cfg')
