--- build.nzbToMedia.git/nzbToSickBeard.py	2016-10-31 22:46:07.203693428 -0500
+++ patches/nzbToSickBeard.py	2016-10-31 23:42:00.768322857 -0500
@@ -43,7 +43,7 @@
 #sbhost=localhost
 
 # SickBeard port.
-#sbport=8081
+#sbport=8082
 
 # SickBeard username.
 #sbusername=
@@ -64,7 +64,7 @@
 # SickBeard watch directory.
 #
 # set this to where your SickBeard completed downloads are.
-#sbwatch_dir=
+#sbwatch_dir=/storage/downloads/tvshows
 
 # SickBeard fork.
 #
