--- build.nzbToMedia.git/nzbToHeadPhones.py	2016-10-31 22:46:07.203693428 -0500
+++ patches/nzbToHeadPhones.py	2016-10-31 23:39:17.863515056 -0500
@@ -41,7 +41,7 @@
 #hphost=localhost
 
 # HeadPhones port.
-#hpport=8181
+#hpport=8084
 
 # HeadPhones uses ssl (0, 1).
 #
@@ -56,7 +56,7 @@
 # HeadPhones watch directory.
 #
 # set this to where your HeadPhones completed downloads are.
-#hpwatch_dir=
+#hpwatch_dir=/storage/downloads/music
 
 # HeadPhones wait_for
 #
