--- build.nzbToMedia.git/nzbToSickBeard.py	2017-02-06 11:43:40.045604021 -0600
+++ patches/nzbToSickBeard.py	2017-02-06 11:58:10.805921888 -0600
@@ -48,7 +48,7 @@
 #sbhost=localhost
 
 # SickBeard port.
-#sbport=8081
+#sbport=8082
 
 # SickBeard username.
 #sbusername=
@@ -69,7 +69,7 @@
 # SickBeard watch directory.
 #
 # set this to where your SickBeard completed downloads are.
-#sbwatch_dir=
+#sbwatch_dir=/storage/downloads/tvshows
 
 # SickBeard fork.
 #
