--- build.nzbToMedia.git/nzbToMedia.py	2016-10-31 22:46:07.203693428 -0500
+++ patches/nzbToMedia.py	2016-10-31 23:47:05.489833890 -0500
@@ -46,7 +46,7 @@
 #cpshost=localhost
 
 # CouchPotato port.
-#cpsport=5050
+#cpsport=8083
 
 # CouchPotato uses ssl (0, 1).
 #
@@ -91,7 +91,7 @@
 #sbhost=localhost
 
 # SickBeard port.
-#sbport=8081
+#sbport=8082
 
 # SickBeard username.
 #sbusername=
@@ -112,7 +112,7 @@
 # SickBeard watch directory.
 #
 # set this if SickBeard and nzbGet are on different systems.
-#sbwatch_dir=
+#sbwatch_dir=/storage/downloads/tvshows
 
 # SickBeard fork.
 #
@@ -198,7 +198,7 @@
 #hphost=localhost
 
 # HeadPhones port.
-#hpport=8181
+#hpport=8084
 
 # HeadPhones uses ssl (0, 1).
 #
@@ -521,6 +521,7 @@
 from core.nzbToMediaUserScript import external_script
 from core import logger, nzbToMediaDB
 
+sys.path.append('/storage/.kodi/addons/script.module.audo-dependencies/lib')
 
 # post-processing
 def process(inputDirectory, inputName=None, status=0, clientAgent='manual', download_id=None, inputCategory=None, failureLink=None):
